src/Core/Services/Implementations/OrganizationService.cs

```diff
@@ -2083,7 +2083,7 @@ private async Task ValidateOrganizationCustomPermissionsEnabledAsync(Guid organi
 
     private async Task<bool> ValidateCustomPermissionsGrant(Guid organizationId, Permissions permissions)
     {
-        if (permissions == null || await _currentContext.OrganizationOwner(organizationId) || await _currentContext.OrganizationAdmin(organizationId))
+        if (permissions == null || await _currentContext.OrganizationAdmin(organizationId))
```

Comment:
This is an optimization to take advantage of the cascading nature of these checks, right? `OrganizationAdmin` includes owners and providerUsers?

Reply:
That's correct! Checking for an explicit `OrganizationOwner` is unnecessary when paired with an `OrganizationAdmin` check because the latter also checks for owner/providers user level permissions.